Cargo.toml

```diff
@@ -4,10 +4,11 @@ authors = ["Eirik Albrigtsen <sszynrae@gmail.com>",
 description = "A simple log implementation that logs to stdout and stderr with colors"
 documentation = "http://clux.github.io/loggerv"
 keywords = ["log", "logger", "stdio", "cli"]
+categories = ["command-line-utilties"]
```

Comment:
i guess there's no better category in the [category_slugs](https://crates.io/category_slugs) atm. maybe `filesystem` could be argued for based on this being IO (to `/dev/std{out/err}`), but this is probably fine.